Earthfile

```diff
@@ -374,10 +375,18 @@ earthly-integration-test-base:
   mirrors = [\"$DOCKERHUB_MIRROR\"]"
 
         IF [ "$DOCKERHUB_MIRROR_INSECURE" = "true" ]
+            IF [ "$DOCKERHUB_MIRROR_HTTP" = "true" ]
+                RUN --no-cache echo "DOCKERHUB_MIRROR_INSECURE and DOCKERHUB_MIRROR_HTTP are mutually exclusive" && exit 1
```

Comment:
technically not true -- a user might have a server that runs a mirror that accepts either. Maybe we should allow it?

Reply:
My own two cents: I think it should be allowed.  At the very least, I've been running with both true on my host (not just the early test setup) since day <span>#</span>1 with no ill effect.  I'd trust the user to know what they're doing and understand the ramifications (even if they actually don't 😄).

If upstream decides to disallow that combo, then I do think your enforcement is a good thing, simply because the the buildkit error would likely be hidden away.